### PR TITLE
Quantity limit is not working properly

### LINKED ISSUE #38: [Bug] Quantity limit is not working properly
Quantity limit is not working properly when registering

The quantity limit is not checked each time a task is added.
Therefore, when the number limit is 5, you can add 100 tasks even if you add 100 tasks.

image
image

### PR DESCRIPTION
**Pull request by kamontia**
Fixed up bug.

Some PR is remaining, so we can't confirm this improvement easily.
After some PR is merged into master, I will remove `WIP`.

close #38

### CODE REVIEW

**Comment by kamontia** [comment]
I resolved this issue.

@chaspy  Please check the image focusing on `kimetaro-dev1`.
image
image

File: kimetaro.py [hunk 1]
```diff
@@ -128,17 +119,32 @@ def add(message):
         Expected pattern:`/add string1 string2`
         '''
         for v in item:
-            LIST[message.channel.id].append(v)
-            added_list.append(v)
+            if canAppend(message):
```
Comment (chaspy) [inline kimetaro.py]: This if/else condition is duplicated.
Could you make it method?
i.e. add_item()

Comment (kamontia) [reply kimetaro.py]: Hey, I updated.
Could you please review and merge?

**Review by chaspy — APPROVED** [review]
LGTM 👍

**Pull request merged**